TestFoundation/TestCalendar.swift

```diff
@@ -131,6 +132,30 @@ class TestCalendar: XCTestCase {
         XCTAssertEqual(components.day, 18)
 
     }
+    
+    func test_gettingDatesOnJapaneseCalendar() {
+        var calendar = Calendar(identifier: .japanese)
+        calendar.timeZone = TimeZone(identifier: "UTC")!
+        
```

Comment:
Could you make this test function a ` throws` and use `TimeZone(identifier: "UTC").unwrapped()` instead of force-unwrappy, we are trying to avoid adding anymore force unwrapping into new tests, thanks